modules/effects/src/effect_notification.ts

```diff
@@ -30,8 +30,19 @@ export function reportInvalidActions(
   }
 }
 
+const typeOf = (value: any) =>
```

Comment:
Generally these should be structured as function declarations, e.g.
```ts
function typeOf(value: any) {...}
```

Reply:
The `typeOf ` is removed. We use `typeof xxx` as you suggested.